make/k8s.sh

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env bash
```

Comment:
technically this is not k8s only, it is infra_cloud module, naming convention should be 

infra_aws for example, k8s_auth function should be separate of this in my opinion and referenced by modules that do use kubernetes provider, please refer to https://github.com/tetrateio/tetrate-service-express-sandbox/blob/main/make/infra_aws.sh and https://github.com/tetrateio/tetrate-service-express-sandbox/blob/main/make/tetrate_managementplane.sh for example

Reply:
ack, adopted to some extent.